wiki/Game_modifier/Fade_Out/id.md

```diff
@@ -0,0 +1,37 @@
+---
+stub: true
+tags:
+  - fade out
+  - fade-out
+  - FO
+  - mod
+  - game modifier
+---
+
+<!-- TODO:
+- add score multiplier, shortcut key, and caption values
+- Add information on when and why the Fade Out mod was removed -->
+
+# Fade Out (mod)
+
+![Ikon mod FO](/wiki/shared/mods/FO.png "Ikon mod Fade Out (FO)")
+
+*Untuk melihat daftar keseluruhan dari semua mod, kunjungi: [Game Modifier](/wiki/Game_modifier)*\
+*Berbeda konteks dengan [Fade In (mod)](/wiki/Game_modifier/Fade_In) atau [Hidden (mod)](/wiki/Game_modifier/Hidden).*
+
+## Tentang
+
+- Singkatan: FO
+- Jenis: Peningkat Kesulitan
+- Tombol shortcut: `F` `F` atau `Shift`+`F`
+- Mode permainan yang cocok: ![][o!m]
+
+## Deskripsi
+
+*Perhatian: Mod ini sudah tidak lagi tersedia untuk digunakan.*
```

Comment:
don't think `Perhatian` is the correct term here

i'll use `Keterangan` instead

Reply:
i'll use `Catatan:` here